test/fixtures/template.html.erb

```diff
@@ -0,0 +1 @@
+<p>Hello.</p>
```

Comment:
Since this https://github.com/bf4/active_model_serializers/commit/6508637ee9bb0e6459e2bdfc6499771cdd29fc61 test was removed I think we can remove this file.